gitbootcamp.rst

```diff
@@ -253,14 +253,21 @@ Compare to the ``main`` branch::
 
    git diff main
 
-Exclude generated files from diff (note the single quotes)::
+Exclude generated files from diff using an `attr <_gitattributes>`_
+`pathspec <_pathspec>`_ (note the single quotes)::
 
    git diff main ':(attr:!generated)'
 
 Exclude generated files from diff by default::
 
    git config diff.generated.binary true
 
+The `generated` `attribute <_gitattributes>`_ is defined in `.gitattributes`,
```

Comment:
reST doesn't like single backticks when it's not a role or link:
```suggestion
The ``generated`` `attribute <_gitattributes>`_ is defined in ``.gitattributes``,
```

I also don't think we need to repeat the link to the gitattributes docs:

```suggestion
The ``generated`` attribute is defined in ``.gitattributes``,
```